### PR TITLE
Created container for SprintsAPI

### PR DESCRIPTION
**Pull request by Bruno-Israel-Mendoza-Aguilar**
SpringBoot container for SprintsAPI for User_Story_07: Create Container for SprintsAPI

### CODE REVIEW

File: src/main/java/com/sprints/Services/Services.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.sprints.Services;
+
+public class Services {
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/sprints/Services/Services.java]: We should rename this class like "SprintsService".

File: src/main/java/com/sprints/controllers/Controllers.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.sprints.controllers;
+
+public class Controllers {
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/sprints/controllers/Controllers.java]: Same scenario here, We should rename this class like "SprintsController".

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+.springBeans
+.sts4-cache
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+
+### NetBeans ###
```
Comment (VictorMunguiaEnciso) [inline .gitignore]: I guess that we should delete the NetBeans ignores on this file.

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+build/
+
+### VS Code ###
```
Comment (VictorMunguiaEnciso) [inline .gitignore]: We should remove the VS code part too.

File: src/main/java/com/sprints/model/Sprints.java [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package com.sprints.model;
+
+public class Sprints {
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/sprints/model/Sprints.java]: This model should be rename like "Sprint".

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.4.RELEASE</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.sprints</groupId>
+	<artifactId>SprintsAPI</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>SprintsAPI</name>
```
Comment (VictorMunguiaEnciso) [inline pom.xml]: Could you please change the name to "sprints" ?

we will try to have a standard in the names with lower case.

File: src/main/resources/application.properties [hunk 7]
```diff
@@ -0,0 +1,4 @@
+server.port=5000
+spring.data.mongodb.uri=mongodb+srv://db_user:[password]j@cluster0-nomfh.mongodb.net/test?retryWrites=true&w=majority
```
Comment (Internship4thSource) [inline src/main/resources/application.properties]: You will not be connecting to the "`test`" database, the correct database name is "`InternHome`".

File: pom.xml [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.4.RELEASE</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.sprints</groupId>
+	<artifactId>sprints</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
```
Comment (Internship4thSource) [inline pom.xml]: Please change the version to 1.0.0-SNAPSHOT

File: src/main/resources/application.properties (hunk 7 above)
Comment (Bruno-Israel-Mendoza-Aguilar) [reply src/main/resources/application.properties]: Done

File: pom.xml (hunk 8 above)
Comment (Bruno-Israel-Mendoza-Aguilar) [reply pom.xml]: Done

**Pull request merged**